bobtimus/src/loan.rs

```diff
@@ -562,7 +589,7 @@ mod tests {
         // The `liquidation_price` is calculated by :
         // `(repayment_amount / current_ltv) / collateral_amount`
         // or with numbers in this example:
-        // (10_500 / 0.75 ) / 1.5 = 9333.33333333
+        // (10000 / 0.75 ) / 1.5 = 9333.33333333
```

Comment:
Why is the repayment amount `10000` now, if the `principal` and `interest_rate` did not change? 
That cannot be correct - with a `principal` of `10,000` and an interest rate of `5%` the `repayment_amount` should still be `10,500` - or am I missing something?